mne/epochs.py

```diff
@@ -3837,16 +3837,17 @@ def equalize_epoch_counts(epochs_list, method="mintime"):
     ----------
     epochs_list : list of Epochs instances
         The Epochs instances to equalize trial counts for.
-    method : str
-        If 'truncate', events will be truncated from the end of each event
-        list. If 'mintime', timing differences between each event list will be
-        minimized.
+    method : ``'truncate'`` | ``'mintime'`` | ``'random'``
+        If ``'truncate'``, events will be truncated from the end of each event
+        list. If ``'mintime'``, timing differences between each event list will be
+        minimized. If ``'random'``, events will be randomly selected from each event
+        list.
 
     Notes
     -----
-    This tries to make the remaining epochs occurring as close as possible in
-    time. This method works based on the idea that if there happened to be some
-    time-varying (like on the scale of minutes) noise characteristics during
+    The method ``'mintime'`` tries to make the remaining epochs occurring as close as
+    possible in time. This method works based on the idea that if there happened to be
```

Comment:
I think "works based on the idea that" is not quite right here ... Maybe "is motivated by the possibility that" would be better?

Reply:
Done